account_move_batch_validate/account.py

```diff
@@ -25,12 +25,16 @@
 from openerp.osv import fields, orm
 from openerp.tools.translate import _
 
-from openerp.addons.connector.queue.job import job
-from openerp.addons.connector.session import ConnectorSession
-from openerp.addons.connector.queue.job import OpenERPJobStorage
-
 _logger = logging.getLogger(__name__)
 
+try:
+    from openerp.addons.connector.queue.job import job
+    from openerp.addons.connector.session import ConnectorSession
+    from openerp.addons.connector.queue.job import OpenERPJobStorage
+except ImportError:
+    _logger.debug('Can not `import connector`.')
```

Comment:
That wasn't the discussion I was referring to, but nevermind, if now we decide this way, we can change it, but contributing guidelines should be updated accordingly.

I'm trying to remember the reasons: it can be any problem with testing inside buildout recipes or something similar?